### ISSUE TITLE
Missing `ndless_resources.tns` on Nspire CX II CAS

### ISSUE THREAD

**Issue by JeffersonDing** [ndless-nspire/Ndless#343]
Currently I have both the installer and the resource file in a folder called `ndless` but when I click on the installer, it fails and says missing the resources file. I'm sure I have it in the same folder.
![04-20-2a022 Image002](https://user-images.githubusercontent.com/51764604/164150452-5e88e2d5-9f41-4a7e-a78b-de40f10e8a4c.jpg)
![04-20-20221 Image001](https://user-images.githubusercontent.com/51764604/164150457-5ab8ea5e-376d-4887-a28e-75cfbef4db59.jpg)
![04-20-art2022 Image003](https://user-images.githubusercontent.com/51764604/164150524-c48472c9-638d-4dc4-af70-abd9ae4b99d0.jpg)

**Comment by Vogtinator** [comment]
Yeah, that looks good. Did you try running the installer directly after a reboot?

**Comment by bananak1n** [comment]
Is support being worked on for 5.5.5.79

**Comment by Ignotius-1** [comment]
Hey, did you ever figure this out? I have the same problem

**Comment by asolidtime** [comment]
I've got the same problem on 5.3.0.564

**Comment by logantischler0** [comment]
Just reboot your calc before trying it again

**Comment by buzz-lightsnack-2007** [comment]
Rebooting is a solution that is personally confirmed. Additionally, at least in V5.2.0.771 running on a Ti-nspire CX II, it seems that the issue more likely occurs when the device is booted after having been shut down for some hours or days, not when the device has been shut down for only a few minutes. Within this time period, the configuration appears to be unset, so even the system time is affected; this may be observed through KhiCAS as long as the clock has not already been altered, where a booted device with clock on or after 12:00 has a possibility to have lost system configuration, and those that are before or hours after are more likely not. Considering this, it may also appear that its recognition of the file might be related to the presence of these variables. 

On another note, the issue appears to also be solved through momentarily putting the device to sleep. The exploit will particularly apply a few moments after waking the system, as long as the document is active. However, I am uncertain whether other versions or hardware supports this alternative method. Could anyone confirm?

**Comment by calc1234** [comment]
I'll try it out with the ti nspire cx 4.5.5.79 hopefully it works :)

**Comment by Zewbra** [comment]
Ndless for 4.5.5.79 isn't supported🙏

**Comment by sasdallas** [comment]
stale
